sdks/python/apache_beam/ml/transforms/handlers.py

```diff
@@ -517,3 +517,8 @@ def expand(
           | "ConvertToRowType" >>
           beam.Map(lambda x: beam.Row(**x)).with_output_types(row_type))
       return transformed_dataset
+
+  def with_exception_handling(self):
+    raise NotImplementedError(
+        "with_exception_handling is not supported for tensorflow-transform "
+        "data processing transforms.")
```

Comment:
It might be good to add a sentence along the lines of `If you want to use exception handling with other MLTransform operations, separate them into a different MLTransform instance`